src/reporters/apiary-reporter.coffee

```diff
@@ -242,21 +238,18 @@ class ApiaryReporter
       else
         return callback error, req, null
 
-    if @configuration.apiUrl?.indexOf('https') is 0
-      if @verbose
-        logger.log 'Starting REST Reporter HTTPS Request'
+    logger.verbose('Starting Apiary reporter HTTP request')
+    if @configuration.apiUrl.match(/^https/)
```

Comment:
When compared with previous version `?` is missing. Please consider, whether it should be present or not.